pkg/deployments/deployments.go

```diff
@@ -0,0 +1,22 @@
+package deployments
```

Comment:
How about moving this under `pkg/k8sutil` instead of creating a new package?
```suggestion
package k8sutil
```